R/bambu-extendAnnotations-utilityExtend.R

```diff
@@ -227,6 +282,12 @@ addNewSplicedReadClasses <- function(combinedTranscriptRanges,
     compatibleQhits[!duplicated(compatibleQhits)]] <- "compatible"
   classificationTable$compatible[classificationTable$equal == "equal"] <- ""
   # annotate with transcript and gene Ids
+  equalShits <- subjectHits(ovExon[mcols(ovExon)$equal])
```

Comment:
this line is repeated below

Reply:
Removed the 2nd line, and refactored S[ubject]hits a29d08cd5e50a191fb7568b38ea0b45970530706